src/recipient_dashboard/tests.py

```diff
@@ -27,14 +27,14 @@ def setUp(self):
 
         # Create active donations for testing
         self.donation1 = Donation.objects.create(
-            organization_id=self.organization,
+            organization_id=self.organization.organization_id,
```

Comment:
Nit here - Same thing here, we probably want this to be in a separate PR in the future !

Reply:
Sure Max, we need objective reviews like these!